examples/helper-script/pom.xml

```diff
@@ -36,7 +36,7 @@
 		<jenkins.workflow.cps.version>2.36</jenkins.workflow.cps.version>
 		<jenkins.workflow.step.version>2.10</jenkins.workflow.step.version>
 		
-		<jenkins-spock.version>2.0.1</jenkins-spock.version>
+		<jenkins-spock.version>2.1.3-SNAPSHOT</jenkins-spock.version>
```

Comment:
These are deliberately at "wrong" versions - SNAPSHOTS - to verify that the release fixes them up to the correct released version.